Form-Controls/index.html

```diff
@@ -14,15 +14,53 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label> Name
```

Comment:
Typically, labels should be attached to an input explicitly -- how might you do this?

Reply:
<div> 
<label for = "name" > name: < /label>
<input type="text" id="name">
<div>